spatial/workerflags_testgym_config.json

```diff
@@ -1,67 +1,54 @@
 {
-  "template": "w2_r0500_e5",
-  "world": {
-    "dimensions": {
-      "x_meters": 10000,
-      "z_meters": 10000
-    },
-    "chunk_edge_length_meters": 50,
-    "legacy_flags": [
-      {
-        "name": "bridge_qos_max_timeout",
-        "value": "0"
-      },
-      {
-        "name": "bridge_soft_handover_enabled",
-        "value": "false"
-      }
-    ],
-    "snapshots": {
-      "snapshot_write_period_seconds": 0
-    }
-  },
-  "load_balancing": {
-    "layer_configurations": [
-      {
-        "layer": "UnrealWorker",
-        "rectangle_grid": {
-          "cols": 1,
-          "rows": 1
-        },
-        "options": {
-          "manual_worker_connection_only": true
-        }
-      }
-    ]
-  },
-  "workers": [
-    {
-      "worker_type": "UnrealWorker",
-      "flags": [
+    "runtime_flags": [],
+    "workers": [
         {
-          "name": "test",
-          "value": "5"
-        }
-      ],
-      "permissions": [
+            "worker_type": "UnrealWorker",
+            "flags": [
+                {
+                  "name": "test",
+                  "value": "15"
+                }
+            ],
+            "permissions":
+            {
+                "entity_creation": true,
+                "entity_deletion": true,
+                "disconnect_worker": true,
+                "reserve_entity_id": true,
+                "entity_query": true
+            },
+            "load_balancing":
+            {
+                "rectangle_grid":
+                {
+                    "cols": 1,
+                    "rows": 1
+                },
+                "manual_worker_connection_only": true
+            }
+        },
         {
-          "all": {}
+            "worker_type": "UnrealClient",
+            "flags": [
+                {
+                  "name": "test",
+                  "value": "5"
```

Comment:
Aha I guess this is the default value, you should link to this line in the user manual to make the connection clear.

Reply:
I think referencing the file from the USER_MANUAL should be adequate. I think referencing lines numbers has proven to be fragile in the past.